intellij-plugin-structure/structure-intellij/src/main/resources/com/jetbrains/plugin/structure/intellij/problems/plugin-problems.json

```diff
@@ -4,7 +4,9 @@
     "com.jetbrains.plugin.structure.intellij.problems.TemplateWordInPluginId": "ignore",
     "com.jetbrains.plugin.structure.intellij.problems.TemplateWordInPluginName": "ignore",
     "com.jetbrains.plugin.structure.intellij.problems.InvalidUntilBuildWithJustBranch": "unacceptable-warning",
-    "com.jetbrains.plugin.structure.intellij.problems.InvalidUntilBuildWithMagicNumber": "unacceptable-warning"
+    "com.jetbrains.plugin.structure.intellij.problems.InvalidUntilBuildWithMagicNumber": "unacceptable-warning",
+    "com.jetbrains.plugin.structure.intellij.problems.ReleaseVersionWrongFormat": "warning",
+    "com.jetbrains.plugin.structure.intellij.problems.ReleaseVersionAndPluginVersionMismatch": "warning"
```

Comment:
Let's have `ReleaseVersionAndPluginVersionMismatch` as an `error` for existing plugins also, and then we'll remap it on the Marketplace side?